### DISCUSSION TITLE
Dataset which contains all bonds

### DISCUSSION THREAD

**Discussion by shrimonmuke0202** [rdkit/rdkit#5347] [Q&A]
Hii all,
I want a database that contains canonical smiles which has these bonds QUADRUPLE, QUINTUPLE, HEXTUPLE, DATIVE, IONIC, and HYDROGEN. Can you give me these types of databases?

**Comment by greglandrum** [comment]
As discussed in your previous question, it's not possible to represent those bond types in SMILES (and I'm not aware of any extensions which do support them), so such a database cannot exist.
